src/content/docs/baas/Firebase/firebase-case-study-nikolaos-digalakis.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Firebase
+keywords: <add some tags>
```

Comment:
@NikosDig This still does not look good to me. You just committed the template I gave you without appropriate edit. 

```
title: Quick Task a case study of Firebase usage
keyword: Firebase, QucikTask
tags: Firebase, QucikTask
```